### ISSUE TITLE
[BUG] `Request body` of a Graphql POST API call is not displayed in the trace viewer

### ISSUE THREAD

**Issue by amalsgit** [microsoft/playwright#15031]
**Context:**
- Playwright Version: 1.22.2
- Operating System: Mac 12.4
- Node.js version: v16.14.2

**Code Snippet**

```javascript
import test from "@playwright/test";

test("sample", async ({ request }) => {
  await request.post("https://swapi.skyra.pw/", {
    data: {
      query: `{
  getFilm(film: 1) {
    director
    episodeId
    openingCrawl
  }
}`,
    },
  });
});

```

**Describe the bug**

`Request body` of a Graphql POST call is not displayed in the trace viewer.

![image](https://user-images.githubusercontent.com/22020329/174979994-1a2e9f06-381b-4d6b-8a07-5ee3ab00b1dc.png)

Can the request query/mutation be also displayed in the trace viewer?

**Comment by mxschmitt** [comment]
I was able to repro it, no request nor response body gets shown inside the network panel of the trace viewer.

**Comment by amalsgit** [comment]
Hi @mxschmitt, thanks for checking from your end. I do see the `Response Body` though as shown in the screenshot. It's the `Request Body` that is missing for me.

**Comment by yury-s** [comment]
Looks like the issue was fixed by #14994, I'll add a test so that it doesn't regress.
